measures/measures-schema.yaml

```diff
@@ -15,7 +15,7 @@ definitions:
         enum: [ia, quality, aci, cost]
       metricType:
         description: Type of measurement that the measure requires in order to attest.
-        enum: [boolean, proportion, singlePerformanceRate, multiPerformanceRate, continuous]
+        enum: [boolean, proportion, singlePerformanceRate, multiPerformanceRate, continuous, float]
```

Comment:
we might want a CAHPS-specific metricType, since there's additional information beyond the float that needs to be gathered - maybe you and @samskeller (who is working on adding the new value type) can decide on a name? 

this is a weak reference to the measurement types published at https://cmsgov.github.io/qpp-submissions-docs/schemas# and we haven't finalized the `continuous` metricType, so this doesn't need to be final